sumo/cli/phonon_bandplot.py

```diff
@@ -114,6 +114,7 @@ def phonon_bandplot(filename, poscar=None, prefix=None, directory=None,
             A -> Z will be used instead.
         eigenvectors (:obj:`bool`, optional): Write the eigenvectors to the
             yaml file.
+        dos_file (str): Path to Phonopy total dos .dat file
```

Comment:
Very nit picky, but we should use the extended sphinx markup for optional function variables: e.g:

```
dos_file (:obj:`str`, optional): Etc.
```